.github/workflows/_publish.yml

```diff
@@ -30,6 +30,40 @@ jobs:
       with:
         sha: ${{ inputs.sha }}
         workflows: ${{ inputs.workflows }}
+    - name: Workflow Status Summary
+      run: |
+        echo "## Publish Workflow Triggered" >> $GITHUB_STEP_SUMMARY
+        echo "" >> $GITHUB_STEP_SUMMARY
+        echo "**Commit SHA:** \`${{ inputs.sha }}\`" >> $GITHUB_STEP_SUMMARY
+        echo "**Conclusion:** ${{ inputs.conclusion }}" >> $GITHUB_STEP_SUMMARY
+        echo "**Continue:** ${{ steps.afterall.outputs.continue }}" >> $GITHUB_STEP_SUMMARY
+        echo "" >> $GITHUB_STEP_SUMMARY
+        echo "### Required Workflows Status" >> $GITHUB_STEP_SUMMARY
+        echo "" >> $GITHUB_STEP_SUMMARY
+
+        # Parse and display the status of each workflow
+        if echo '${{ steps.afterall.outputs.status }}' | jq -r '
+          to_entries | .[] |
+          "- **\(.key):** Status: `\(.value.status)`, Conclusion: `\(.value.conclusion)`, Run ID: `\(.value.id)`"
+        ' >> $GITHUB_STEP_SUMMARY 2>/dev/null; then
+          echo "" >> $GITHUB_STEP_SUMMARY
+        else
+          echo "_Status information unavailable_" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+        fi
+
+        echo "### Workflow Run IDs" >> $GITHUB_STEP_SUMMARY
+        echo "" >> $GITHUB_STEP_SUMMARY
+
+        # Display the run IDs being used for artifact downloads
+        if echo '${{ steps.afterall.outputs.run-ids }}' | jq -r '
```

Comment:
we have a jq action for jq - dont want to see jqlang outside of that (bson wraps it)